### PR TITLE
[Bug 1748716] Add scheduled query for stable table column count monitoring

### PR DESCRIPTION
**Pull request by scholtzan**
The query has been migrated from https://sql.telemetry.mozilla.org/queries/76766/source

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

**Comment by scholtzan** [comment]
I executed the query to add some data to `moz-fx-data-shared-prod.monitoring_derived.stable_table_column_counts_v1`. Looks in line with the Redash query.

**Review by ANich — APPROVED** [review]
🚢

File: sql/moz-fx-data-shared-prod/monitoring_derived/stable_table_column_counts_v1/script.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+-- Insert into target table
+INSERT INTO
+  monitoring_derived.stable_table_column_counts_v1
+SELECT
+  DATE(@submission_date) AS submission_date,
+  dataset,
+  table_name,
+  total_columns
+FROM
+  columns
```
Comment (ANich) [inline sql/moz-fx-data-shared-prod/monitoring_derived/stable_table_column_counts_v1/script.sql:53]: Question: how do scripts like this behave when you dry run them? I assume they won't do the insert but any other gotchas?

Comment (scholtzan) [reply sql/moz-fx-data-shared-prod/monitoring_derived/stable_table_column_counts_v1/script.sql:53]: When dry running scripts BigQuery always returns a success state without actually dry running or executing the script. So we also can't get schema information etc from scripts

**Pull request merged**